src/Sulu/Bundle/AdminBundle/Resources/js/components/Radio/Radio.js

```diff
@@ -25,31 +29,45 @@ export default class Radio extends React.PureComponent<Props> {
     };
 
     handleClick = () => this.input.click();
+
     handleInputClick = (event: Event) => event.stopPropagation();
+
     setInput = (input: ElementRef<'input'>) => this.input = input;
 
     render() {
+        const {
+            skin,
+            name,
+            checked,
+            children,
+            className,
+        } = this.props;
+        const radioClass = classNames(
```

Comment:
Nevermind, your way seems to be the recommended one in [their README](https://github.com/JedWatson/classnames#usage-with-reactjs).